challenge_set_a_table/src/challenge_set_a_table_states/config.py

```diff
@@ -0,0 +1,2 @@
+MIN_GRAB_OBJECT_HEIGHT = 0.075
+MAX_GRAB_OBJECT_WIDTH = 0.15
```

Comment:
Please put this in the robocup_knowledge

Reply:
That would mean duplicating it